servers/src/proto/journal_entry_protos.proto

```diff
@@ -0,0 +1,204 @@
+package tachyon.proto;
+option java_package = "tachyon.proto";
+
+// Wraps around all types of Tachyon journal messages.
+// next available id: 25
+message JournalEntry {
+  optional int64 sequence_number = 1;
```

Comment:
Maybe it is here, but is there a good place to comment on how the sequence_number would work? Basically, since proto doesn't have inheritance, every JournalEntry will not have the sequence_number set until right before it is written.